src/stores/database/postgres/ethereum.go

```diff
@@ -99,6 +100,35 @@ func (ea *ETHAccounts) GetAllDeleted(ctx context.Context) ([]*entities.ETHAccoun
 	return accounts, nil
 }
 
+func (s *ETHAccounts) ListAddresses(ctx context.Context, isDeleted bool, limit, offset int) ([]string, error) {
```

Comment:
could this return []common.Address ?

Reply:
I also considered that, but I decided on this so we dont not tight out Data layer to anything else but Entities. Let me know if you think otherwise

Reply:
But common.Address is an entity right? It's just not one of our entities. But if you don't want to be tied to an external package it's fine to me.